config.js

```diff
@@ -20,7 +20,7 @@ module.exports = {
   "host": "0.0.0.0",
   "port": 6833,
   "adminemail": "davidcruz@bitgo.com",
-  "mongouri": "mongodb://localhost:27017/key-recovery-service",
+  "mongouri": process.env.KRSV2_DBURL,
```

Comment:
Why this change? Is the Mongo URI sensitive?

Reply:
yeah its sensitive - the way i'm using mongo now, the uri includes the username and password